src/test/java/com/aws/greengrass/clientdevices/auth/CertificateManagerTest.java

```diff
@@ -86,10 +89,13 @@ public class CertificateManagerTest {
     Path tmpPath;
 
     private CertificateManager certificateManager;
+    private CertificateStore certificateStore;
 
     @BeforeEach
     void beforeEach() throws KeyStoreException {
```

Comment:
Even though `certManager` will go away eventually I think having a test for this is good while we still maintain it.